### PR TITLE
Fix API key usage

### PR DESCRIPTION
**Pull request by leandropineda**
### Description

Fixed RobotSession class so it uses the robot API key for sending status instead of service API key

### CODE REVIEW

**Comment by clarisandev** [comment]
@leandropineda  what a super quick fix! I left a dummy question, if it's ok then LGTM :)

File: inorbit_edge/tests/test_fetch_robot_config.py [hunk 1]
```diff
@@ -40,11 +40,11 @@
 def test_get_robot_config_from_session():
     # test required parameters only
     robot_session = RobotSession(
-        robot_id="id_123", robot_name="name_123", api_key="appkey_123"
+        robot_id="id_123", robot_name="name_123", api_key="apikey_123"
```
Comment (clarisandev) [inline inorbit_edge/tests/test_fetch_robot_config.py:43]: this one shouldn't be "robot_apikey_123"? @leandropineda

**Review by clarisandev — APPROVED** [review]
@leandropineda  what a super quick fix! I left a dummy question, if it's ok then LGTM :)

File: inorbit_edge/tests/test_fetch_robot_config.py (hunk 1 above)
Comment (leandropineda) [reply inorbit_edge/tests/test_fetch_robot_config.py:43]: This is the API key you get from the developer console. It is used for authenticating against InOrbit REST API to then get the `robot_api_key`

**Pull request merged**